R/paste.R

```diff
@@ -168,15 +168,20 @@ dm_paste_select <- function(dm) {
 dm_paste_pks <- function(dm) {
   dm %>%
     dm_get_all_pks_impl() %>%
-    mutate(code = glue("dm::dm_add_pk({tick_if_needed(table)}, {deparse_keys(pk_col)})")) %>%
+    mutate(
+      code = case_when(
+        !is.na(autoincrement) & autoincrement ~ glue("dm::dm_add_pk({tick_if_needed(table)}, {deparse_keys(pk_col)}, autoincrement = TRUE)"),
```

Comment:
How can `autoincrement` be `NA` here?

Reply:
Currently this can happen when learning a dm from a remote data source.